src/helpers/Datasets.ts

```diff
@@ -52,6 +53,11 @@ export class Datasets {
     }
 
     public getDefaults() {
-         return configDefault
+        if (this.type == 'master') {
```

Comment:
Don't hardcode the string and read all the constants from the configs.  Is case sensitive check not required? Also, can we have a strict type comparison?

Reply:
Also, you can simplify it like this - `return defaultConfig[this.type]` without conditions.